src/main/resources/db/migration/V40__migrate_UnacceptableAbsence.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE OFFENDER_ATTENDANCE
+    ALTER COLUMN ABSENT_REASON TYPE VARCHAR(40);
+
+UPDATE OFFENDER_ATTENDANCE
+SET ABSENT_REASON = 'UnacceptableAbsenceIncentiveLevelWarning'
```

Comment:
we'll be adding a new `UnacceptableAbsence` reason shortly, so hence have to rename this one